server/src/utils/request.ts

```diff
@@ -0,0 +1,5 @@
+export const fromChecksum = (checksum: string): Buffer => {
+  return Buffer.from(checksum, checksum.length === 28 ? 'base64' : 'hex');
```

Comment:
Why can it be guaranteed that base64 is always 28 character long? lol

Reply:
https://stackoverflow.com/a/13378842